src/NzbDrone.Core/Parser/Parser.cs

```diff
@@ -455,7 +455,7 @@ public static class Parser
         private static readonly Regex LanguageRegex = new Regex(@"(?:\W|_)(?<italian>\b(?:ita|italian)\b)|(?<german>german\b|videomann)|(?<flemish>flemish)|(?<greek>greek)|(?<french>(?:\W|_)(?:FR|VOSTFR)(?:\W|_))|(?<russian>\brus\b)|(?<dutch>nl\W?subs?)|(?<hungarian>\b(?:HUNDUB|HUN)\b)|(?<spanish>\b(?:español|castellano)\b)",
                                                                 RegexOptions.IgnoreCase | RegexOptions.Compiled);
 
-        private static readonly Regex YearInTitleRegex = new Regex(@"^(?<title>.+?)(?:\W|_)?(?<year>\d{4})",
+        private static readonly Regex YearInTitleRegex = new Regex(@"^(?<title>.+?)[-_. ]+?\(?(?<year>\d{4})\)?",
```

Comment:
Now that the regex is more precise, it will only handle separator then year (e.g. `Series 2021` or `Series.2021`) or separator then parenthesis then year (e.g. `Series (2021)`). 
@markus101, are there any other characters that might enclose year (e.g `[`) that we need to cover?

Reply:
I'm sure someone will come up with some wild naming for some release, but not concerned with that.